indigo_api_callback.h

```diff
@@ -234,7 +234,9 @@ struct tlv_to_config_name* find_generic_tlv_config(int tlv_id, struct tlv_to_con
 static int get_control_app_handler(struct packet_wrapper *req, struct packet_wrapper *resp);
 static int start_loopback_server(struct packet_wrapper *req, struct packet_wrapper *resp);
 static int stop_loop_back_server_handler(struct packet_wrapper *req, struct packet_wrapper *resp);
+#ifndef CONFIG_ZEPHYR
```

Comment:
This can be a upstream commit with a feature flag. Actually, why do we even need to protect the header file? Just no implementing them should be fine, no?

Reply:
> This can be a upstream commit with a feature flag. Actually, why do we even need to protect the header file? Just no implementing them should be fine, no?

I have removed full API functionality, so compilation errors are seen. So protected declarations under Kconfig.

error: 'start_ap_handler' declared 'static' but never defined [-Werror=unused-function]

Reply:
As we discussed many times, our goal is to minimize changes to OSS parts, so, better way is to define dummy functions.

Reply:
Can I just define AP related API's as TODO functions?